pilot/pkg/networking/core/v1alpha3/cluster_builder.go

```diff
@@ -1286,3 +1287,35 @@ func defaultTransportSocketMatch() *cluster.Cluster_TransportSocketMatch {
 		TransportSocket: xdsfilters.RawBufferTransportSocket,
 	}
 }
+
+// generates a cluster that acts as external SDS server
+func (cb *ClusterBuilder) buildExternalSDSCluster(addr string) *cluster.Cluster {
+	addr = strings.TrimPrefix(addr, authn_model.UnixAddressPrefix)
```

Comment:
Similar comment - if you use unix socket and agent proxy, this is already done.
Otherwise - it should be a cluster with full destination rules applied, etc - i.e. normal code path.